Project2015To2017.Core/Project2015To2017.Core.csproj

```diff
@@ -1,8 +1,9 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Encoding? BOM? Non-blocking.

Reply:
Nothing intentional, I wonder if Visual Studio has started adding in BOM by default or something. See what you think of my other PR as I think that would then remove all these changes from here if we merge it first.